### DISCUSSION TITLE
how to create a video/gif of point cloud?

### DISCUSSION THREAD

**Discussion by wiesehahn** [r-lidar/lidR#673] [Q&A] [answered]
What is a good way to create a video/gif for example of a rotating point cloud?

E.g. how was the `point-cloud-rotating.gif` in the Readme created?

**Comment by Jean-Romain** [comment, marked as answer]
see https://stackoverflow.com/questions/64046462/making-nice-gif-from-lidar-point-cloud-with-r-using-lidr-package

**Reply by wiesehahn** [reply]
wow, I am always amazed by your instant replies. Seems like a bot was implemented ;)

Thanks!

**Reply by Jean-Romain** [reply]
I swear I'm a real human. You were lucky on that one :wink:
